napari/layers/_image_layer/model.py

```diff
@@ -312,17 +298,16 @@ def method(self, method):
     def interpolation(self):
         """string: Equipped interpolation method's name.
         """
-        return _index_to_name(self.interpolation_index)
+        return str(self._interpolation)
```

Comment:
IMO this should just be a wrapper around `self._node.interpolation` which calls into question if we really need an Enum at all.

Reply:
I like that the Enum allows us more control over which modes we expose and don't (i.e. it lets us drop `sinc`)